code/game/machinery/doors/brigdoors.dm

```diff
@@ -303,40 +355,72 @@
 // 	"change" resets the timer to the timetoset amount while the timer is counting down
 // Also updates dialog window and timer icon
 /obj/machinery/door_timer/Topic(href, href_list)
-	if(..())
-		return 1
+	..()
 
 	if(!allowed(usr) && !usr.can_admin_interact())
 		return 1
 
 	usr.set_machine(src)
 
-	if(href_list["timing"])
-		timing = text2num(href_list["timing"])
-
-		if(timing)
-			timer_start()
+	if(href_list["timingstart"])
+		if(!detaineecrimes | isnull(detaineename) | timetoset == 0)
+			visible_message("<span class='warning'>The [src] buzzes. Cannot activate without all information inputted.</span>")

```

Comment:
You *shouldn't* need the `The`

Might better grammatically as `without all required information.`